styles/Middle.module.css

```diff
@@ -1,208 +1,223 @@
-    color: #565959;
-    font-weight: 400;
+  font-size: 24px;
+  line-height: 32px;
+  color: #565959;
+  font-weight: 400;
 }
 
 .ratings {
-    display: flex;
-    flex-direction: row;
-    border-bottom: 1px solid #e7e7e7;
+  display: flex;
+  flex-direction: row;
+  border-bottom: 1px solid #e7e7e7;
 }
 
 .stars {
-    height: 18px;
-    width: 80px;
+  height: 18px;
+  width: 80px;
 }
 
 .starsPic {
-    background-image: url(https://m.media-amazon.com/images/S/sash/ZNt8quAxIfEMMky.png);
-    background-position: -2px -42px;
-    height: 18px;
-    width: 80px;
+  background-image: url(https://m.media-amazon.com/images/S/sash/ZNt8quAxIfEMMky.png);
+  background-position: -2px -42px;
+  height: 18px;
+  width: 80px;
 }
 
 .numRatings {
-    padding-left: 15px;
+  padding-left: 15px;
 }
 
 .symbol {
-    font-size: 13px;
-    vertical-align: top;
-    line-height: 9.5px;
+  font-size: 13px;
+  vertical-align: top;
+  line-height: 9.5px;
 }
 
 .dollars {
-    font-size: 28px;
+  font-size: 28px;
 }
 
 .cents {
-    font-size: 13px;
-    vertical-align: top;
-    line-height: 9.5px;
+  font-size: 13px;
+  vertical-align: top;
+  line-height: 9.5px;
 }
 
 .learn {
-    vertical-align: top;
-    line-height: 17.5px;
-    padding-left: 3px;
+  vertical-align: top;
+  line-height: 17.5px;
+  padding-left: 3px;
 }
 
-.type, .colorLabel, .sizeLabel {
-    color: #555;
-    padding-left: 2px;
-    padding-bottom: 2px;
-    font-weight: 400;
+.type,
+.colorLabel,
+.sizeLabel {
+  color: #555;
+  padding-left: 2px;
+  padding-bottom: 2px;
+  font-weight: 400;
 }
 
 .genderSelect {
-    list-style: none;
-    display: flex;
-    flex-direction: row;
-    justify-content: flex-start;
-    padding-left: 0px;
-    margin-top: 2px;
+  list-style: none;
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+  padding-left: 0px;
+  margin-top: 2px;
 }
 
 .genderBtn {
-   border: 1px solid #e0e0e0;
-   background-color: transparent;
-   margin-top: 4px;
-   margin-bottom: 4px;
-   margin-left: 6px;
-   font-size: 13px;
-   height: 30px;
-
+  border: 1px solid #e0e0e0;
+  background-color: transparent;
+  margin-top: 4px;
+  margin-bottom: 4px;
+  margin-left: 6px;
+  font-size: 13px;
+  height: 30px;
 }
 
 .colorOptions {
-    list-style: none;
-    display: flex;
-    flex-direction: row;
-    margin-top: 4px;
-    padding-left: 0px;
-    flex-wrap: wrap;
+  list-style: none;
+  display: flex;
+  flex-direction: row;
+  margin-top: 4px;
+  padding-left: 0px;
+  flex-wrap: wrap;
 }
 
 .optionIMG {
-    height: 50px;
-    width: 38px;
-    
+  height: 50px;
+  width: 38px;
+  object-fit: fill;
 }
 
 .optionBtn {
-    padding: 0;
-    background-color: transparent;
-    margin-left: 6px;
-    border: 1px solid #e0e0e0;
+  padding: 0;
+  background-color: transparent;
+  margin-left: 6px;
+  border: 1px solid #e0e0e0;
+  cursor: pointer;
+}
+.optionBtn.active {
+  border-color: #e77600;
+}
+.optionBtn:hover {
+  border-color: #bebebe;
```

Comment:
added cursor pointer to each btn

if the current button is active change the border color, same if it is hovered